src/server/worldserver/worldserver.conf.dist

```diff
@@ -796,7 +797,7 @@ CharactersPerRealm = 12
 #
 #    HeroicCharactersPerRealm
 #        Description: Limit number of heroic class characters per account on this realm.
-#        Range:       1-10
+#        Range:       1-12
 #        Default:     1
```

Comment:
If you check in Shauren's commit, https://github.com/TrinityCore/TrinityCore/commit/79aafd5865f11b5ecdd2865829562b981652db75#diff-cb0c3fe3d6f6147fc1a95217aff13561L800 , you will see that he did not change it either. Maybe you could propose that change in a new PR only for worldserver.conf.dist (unless it is added in one of the upcoming PRs before Legion is live from 2016-08-30).